.env-devel

```diff
@@ -39,6 +39,9 @@ S3_SECURE=0
 
 SMTP_HOST=mail.speag.com
 SMTP_PORT=25
+SMTP_USERNAME=None
+SMTP_PASSWORD=None
+SMTP_TLS_ENABLED=0
```

Comment:
does this work for both tls or ssl securities?

Reply:
the SMTP server lib [supports direct connection via TLS/SSL](https://aiosmtplib.readthedocs.io/en/stable/usage.html#connection-options)